samples/system_time/README.rst

```diff
@@ -0,0 +1,31 @@
+.. _System Time Sample App:
```

Comment:
Since this sample isn't within an existing samples folder (you're creating a new samples/system_time folder), you'll also need to create the index infrastructure and linking for the documentation.  I can do that later if you'd like, so to get around the warning saying this README.rst isn't included in any TOC tree, add a line to the top of the file:

     :orphan: